lms/djangoapps/verify_student/management/commands/tests/test_retry_failed_photo_verifications.py

```diff
@@ -21,16 +23,17 @@
 
 LOGGER_NAME = 'retry_photo_verification'
 
-
 # Lots of patching to stub in our own settings, and HTTP posting
+
```

Comment:
NIT: I don't think these spaces should be here (just because the comment applies to the patches)